packages/@ember/-internals/glimmer/lib/modifiers/action.ts

```diff
@@ -8,12 +8,13 @@ import {
   CapturedNamedArguments,
   CapturedPositionalArguments,
   DynamicScope,
-  ModifierManager,
+  GlimmerTreeChanges,
+  InternalModifierManager,
   VMArguments,
 } from '@glimmer/interfaces';
 import { isInvokableRef, updateRef, valueForRef } from '@glimmer/reference';
 import { registerDestructor } from '@glimmer/runtime';
-import { createUpdatableTag } from '@glimmer/validator';
+import { createUpdatableTag, UpdatableTag } from '@glimmer/validator';
```

Comment:
Not sure how you feel, but I'd love it if type only imports were done separately....

```suggestion
import { createUpdatableTag } from '@glimmer/validator';
import type { UpdatableTag } from '@glimmer/validator';
```

Reply:
I mainly add imports via autocomplete (Command + . in VSCode), so if we can set it up so that it does `type` imports for types, I'm happy with that

Reply:
We also get a linting error for this, so I'm going to leave it as is for now